src/nunit.analyzers/ConstActualValueUsage/ConstActualValueUsageCodeFix.cs

```diff
@@ -71,7 +71,7 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
         }
 
         private static bool TryFindArguments(SemanticModel semanticModel, InvocationExpressionSyntax invocationSyntax,
-            out ExpressionSyntax expectedArgument, out ExpressionSyntax actualArgument)
+            out ExpressionSyntax? expectedArgument, out ExpressionSyntax? actualArgument)
```

Comment:
Sorry I'm late. Is this a case where `[NotNullWhen(true)]` would apply on each argument?

Reply:
Yep, my bad.